src/web/routes.rs

```diff
@@ -21,6 +21,10 @@ pub(super) fn build_routes() -> Routes {
     //   https://developers.google.com/search/reference/robots_txt#handling-http-result-codes
     //   https://support.google.com/webmasters/answer/183668?hl=en
     routes.static_resource("/robots.txt", PermanentRedirect("/-/static/robots.txt"));
+    routes.static_resource(
+        "/googleb54494ba4d52c200.html",
```

Comment:
Where does this string come from? I clicked on the "verification flow" link in the webmaster docs, but didn't see anything that looked relevant.

Reply:
If you visit https://search.google.com/search-console/welcome from a logged-in Google Account, and click the "URL Prefix" box, enter "https://docs.rs", and hit "continue," you'll get a similar string.